redbox/redbox/retriever/retrievers.py

```diff
@@ -86,11 +86,14 @@ def _get_relevant_documents(self, query: str, *, run_manager: CallbackManagerFor
             raise ValueError("OpenSearch client or document mapper is not initialized")
 
         body = self.body_func(query)
-        logger.info("query to opensearch: from get_relevant_documents")
-        logger.info(str(body))
-        response = self.es_client.search(index=self.index_name, body=body)
-        logger.info("response from opensearch: from get_relevant_documents")
-        logger.info(str([self.document_mapper(hit) for hit in response["hits"]["hits"]]))
+        logger.warning("query to opensearch: from get_relevant_documents")
+        logger.warning(str(body))
+        try:
+            response = self.es_client.search(index=self.index_name, body=body)
+        except Exception as e:
```

Comment:
Not logged and not raised, plus generic exception handling not ideal - can we be more specific?